src/next/components/Button/Button.tsx

```diff
@@ -0,0 +1,48 @@
+import { ButtonProps } from "~/common"
+import { SUPPORTED_ICONS_MAP } from "~/common/Button"
+
+// Theme specific config
+const NEXT_BUTTON_COLOR_VARIANTS = ["black", "white"] as const
+export type NextButtonColorVariant = (typeof NEXT_BUTTON_COLOR_VARIANTS)[number]
```

Comment:
wasnt sure if I should define here or in /common because it is theme specific but at the same time I thought would have circular imports. but in the end it worked so I left it here

Reply:
Hmm I think this might be better to put inside `~/common`? We might want to use primary/secondary/inverse in this case and let themes define what primary/secondary/inverse means.